renovate.json

```diff
@@ -1,11 +1,12 @@
 {
-  "enabled": false,
+  "enabled": true,
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["config:base"],
   "prConcurrentLimit": 10,
   "branchConcurrentLimit": 0,
   "branchPrefix": "renovate-",
   "prHourlyLimit": 1,
+  "rebaseWhen": "never",
```

Comment:
>It is also recommended to avoid rebaseWhen=never as it can result in conflicted branches with outdated PR descriptions and/or status checks.

https://docs.renovatebot.com/configuration-options/#rebasewhen

Reply:
@wilsonianb  Should we set it back to auto / not set it?

Reply:
I vote set it to `conflicted` or not set it (which I think will do `conflicted` based on our renovate + repo settings.)
>auto: Renovate will autodetect the best setting. It will use behind-base-branch if configured to automerge or repository has been set to require PRs to be up to date. Otherwise, conflicted will be used instead

(`auto` is the default)

Reply:
```suggestion
  "rebaseWhen": "conflicted",
```